run.go

```diff
@@ -67,20 +70,55 @@ func run(context *cli.Context) error {
 		return errors.New("invalid runtime config")
 	}
 
-	if err := create(context.Args().First(),
+	consolePath := context.String("console")
+
+	var wg sync.WaitGroup
+	var console Console
+	var err error
+	var consoleState *term.State
+
+	// if consolePath is /dev/ptmx or /dev/pts/ptmx
+	// means that we have to allocate a new pts
+	if consolePath == ptmxPath || consolePath == ptsPtmxPath {
```

Comment:
Yup, I do realise that. Some of the discussion happened on IRC and wasn't logged on issues.